package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "club-view",
+  "version": "1.0.0",
+  "description": "ClubView Main Repo",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:frontend": "cd frontend && npm run start",
+    "start:backend": "cd backend && npm run start",
```

Comment:
Let's use `npm workspaces` so we don't have to define all these directory traversals.

https://docs.npmjs.com/cli/v8/using-npm/workspaces
Note which `npm` version you're running - I recommend switching to the current one (v8) especially since this is a fresh project.